homeassistant/components/sun.py

```diff
@@ -168,6 +170,15 @@ def solar_elevation(self):
             self.location.latitude,
             self.location.longitude)
 
+    @property
+    def solar_azimuth(self):
+        """Angle the sun is clockwise from North."""
+        from astral import Astral
+        return Astral().solar_azimuth(
```

Comment:
I see that this is done in a similar way to `solar_elevation` but that's wrong. Both values should be assigned in the `update_as_of` method with the given time. Otherwise the time of the azimuth and elevation could be off!